account_invoice_merge/models/account_invoice.py

```diff
@@ -58,15 +47,16 @@ def _get_first_invoice_fields(self, invoice):
             'state': 'draft',
             'reference': '%s' % (invoice.reference or '',),
             'name': '%s' % (invoice.name or '',),
-            'fiscal_position': invoice.fiscal_position.id,
-            'payment_term': invoice.payment_term.id,
-            'period_id': invoice.period_id.id,
-            'invoice_line': {},
+            'fiscal_position_id': invoice.fiscal_position_id.id,
+            'payment_term_id': invoice.payment_term_id.id,
+            # 'period_id': invoice.period_id.id,
+            'invoice_line_ids': {},
             'partner_bank_id': invoice.partner_bank_id.id,
         }
 
     @api.multi
-    def do_merge(self, keep_references=True, date_invoice=False):
+    def do_merge(self, keep_references=True, date_invoice=False,
+                 remove_empty_invoice_lines=True):
```

Comment:
That feature was added by @alexis-via

A fact is that Is default True so the corresponding part is always executed. But my guess is that it is added as an option for more customization possibilities in other modules.